source/agora/consensus/validation/Transaction.d

```diff
@@ -93,6 +100,10 @@ public string isInvalidReason (
         if (height < utxo_value.unlock_height + input.unlock_age)
             return "Transanction: Input's unlock age cannot be used for this block height";
 
+        if (auto error = engine.execute(utxo_value.output.lock, input.unlock,
+            tx, input))
+            return error;
```

Comment:
The error message should be the same format as others starting with "Transaction: ".

Reply:
It will allocate a string always this way. I wonder how we can avoid this..